src/Parser/MarksmanshipHunter/Modules/Features/AimedInVulnerableTracker.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from "common/format";
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class AimedInVulnerableTracker extends Module {
+  static dependencies = {
+    enemies: Enemies,
+  };
+  totalAimed = 0;
+  inVulnAimed = 0;
+  outsideVulnAimed = 0;
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.AIMED_SHOT.id) {
+      return;
+    }
+    const enemy = this.enemies.getEntity(event);
+    if (enemy.hasBuff(SPELLS.VULNERABLE.id, event.timestamp)) {
+      this.inVulnAimed += 1;
+    }
+    if (!enemy.hasBuff(SPELLS.VULNERABLE.id, event.timestamp)) {
+      this.outsideVulnAimed += 1;
+    }
+    this.totalAimed += 1;
+  }
+  suggestions(when) {
+    const aimedOutsideVuln = this.outsideVulnAimed;
+    when(aimedOutsideVuln).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> You have {this.outsideVulnAimed} <SpellLink id={SPELLS.AIMED_SHOT.id} />s outside <SpellLink id={SPELLS.VULNERABLE.id} />. Try and minimize these, as they deal significantly less damage than their vulnerable counterparts. It should be noted that rarely, you will be casting non-vulnerable aimeds due to no procs and/or focus capping. </span>)
```

Comment:
1) I'd rephrase the beginning as "You have casted X Aimed Shots ..."
2) Considering you mentioned it's possible that a player would have to cast some AS without Vulnerable, wouldn't you think it might be better to reconsider the thresholds for the suggestions? Maybe make them more tolerant? Or divide by fight duration, make them "AS outside Vulnerable *per minute*" and use that?

Reply:
It should really generally always be 100%, because it should be quite rare that you are forced to cast them outside, but I might've been a bit too strict when I made it, so it's a bit more lenient now, to allow for up to 2% of total aimeds to hit outside before it triggers the suggestion